repository/src/main/resources/alfresco/subsystems/Authentication/identity-service/identity-service-authentication-context.xml

```diff
@@ -152,6 +152,9 @@
       <property name="clientIdValidationDisabled">
          <value>${identity-service.client-id.validation.disabled:true}</value>
       </property>
+      <property name="adminConsoleRedirectPath">
+         <value>${identity-service.admin-console.redirect-path}</value>

```

Comment:
This property cannot be null, the default value is already declared in the `identity-service-authentication.properties` file. Similar declaration you can see for the `identity-service.resource` property. Anyway, that value must be declared in the `.properties` file, because we have different values for the community and the enterprise version.